.github/workflows/publish_container.yml

```
@@ -0,0 +1,14 @@
+name: Publish Docker
+on: [push]
```

I've seen other examples of github actions where they restrict branches this is run on. I'm thinking maybe restrict to code pushed to master?
```suggestion
on:
  push:
    branches:
      - master
```

---

we can do that, I had to leave it out to test it